ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -224,19 +224,17 @@
             </v-col>
           </v-row>
 
-          <div class="px-4">
-            <v-divider />
-          </div>
+          <v-divider  v-if="showLandDetails" class="mx-4"/>
 
           <!-- Land Details -->
-          <v-row no-gutters class="px-6 pt-6" >
+          <v-row v-if="showLandDetails" no-gutters class="px-6 pt-6" >
             <v-col cols="3" class="pt-1">
               <h3>Land Details</h3>
             </v-col>
           </v-row>
 
           <!-- Lease or Land Ownership -->
-          <v-row no-gutters class="px-6 pt-1" >
+          <v-row v-if="showLandDetails" no-gutters class="px-6 pt-1" >
```

Comment:
Very minor but since they are all associated to the same condition, you could wrap them all in a conditional template and drop the condition on each element.
ie `<template v-if="showLandDetails"></..>`

Reply:
Yeah I can do that!